README.md

```diff
@@ -1,69 +1,89 @@
 
-By default, the step creates each environment variable name from the secret name, transformed to include only uppercase letters, numbers, and underscores, and so that it doesn't begin with a number. 
+By default, the step creates each environment variable name from the secret name, transformed to include only uppercase letters, numbers, and underscores, and so that it doesn't begin with a number.   
 
-To set the environment variable name, enter it before the secret ID, followed by a comma. For example `ENV_VAR_1, secretId` creates an environment variable named **ENV_VAR_1** from the secret `secretId`. 
+To set the environment variable name, enter it before the secret ID, followed by a comma. For example `ENV_VAR_1, secretId` creates an environment variable named **ENV\_VAR\_1** from the secret `secretId`. 
 
-The environment variable name can consist of uppercase letters, numbers, and underscores.  
+The environment variable name can consist of uppercase letters, numbers, and underscores. 
 
 To use a prefix, enter at least three characters followed by an asterisk. For example `dev*` matches all secrets with a name beginning in **dev**. The maximum number of matching secrets that can be retrieved is 100. If you set the variable name, and the prefix matches multiple secrets, then the action fails.
-​
+
 - `parse-json-secrets`
 
 (Optional - default false) By default, the action sets the environment variable value to the entire JSON string in the secret value. 
 
 Set `parse-json-secrets` to `true` to create environment variables for each key/value pair in the JSON.
 
 Note that if the JSON uses case-sensitive keys such as "name" and "Name", the action will have duplicate name conflicts. In this case, set `parse-json-secrets` to `false` and parse the JSON secret value separately. 
-​
-### Examples
-​
-**Example 1: Get secrets by name and by ARN**  
+
+## Environment variable naming<a name="retrieving-secrets_github_alias"></a>
+
+The environment variables created by the action are named the same as the secrets they comes from. If you parse the JSON of the secret, then the environment variable name includes both the secret name and the JSON key name, for example `MYSECRET_KEYNAME`. 
+
+Environment variables have stricter naming requirements than secrets, so this action transforms secret names to meet those requirements. For example, the action transforms lowercase letters to uppercase letters. Because of the transformed names, two environment variables might end up with the same name. For example, a secret named "MySecret" and a secret named "mysecret" would both become environment variables named "MYSECRET". In this case, the action will fail, because environment variable names must be unique. Instead, you must specify the name you want to use for the environment variable.
+
+You can set the environment variable name by specifying an *alias*, as shown in the following example which creates a variable named `ENV_VAR_NAME`.
+
+```
+secret-ids:
```

Comment:
`secret_ids: |`

We are parsing it as multiline string. Not an object.

Reply:
added in!